package.json

```diff
@@ -29,35 +17,32 @@
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-react": "^7.4.0",
-    "expose-loader": "~0.6.0",
-    "file-loader": "^0.9.0",
+    "expose-loader": "^0.7.4",
     "identity-obj-proxy": "^3.0.0",
     "jest-cli": "^20.0.0",
     "jsdom": "^9.5.0",
     "nock": "^9.0.14",
     "node-sass": "^4.5.0",
-    "react-addons-test-utils": "^15.3.1",
     "react-test-renderer": "^16.0.0",
     "redux-mock-store": "^1.2.2",
-    "sass-loader": "~6.0.6",
-    "style-loader": "^0.13.1",
     "stylelint": "^7.13.0",
     "stylelint-config-standard": "^16.0.0",
-    "url-loader": "^0.5.7",
     "webpack": "^3.4.1",
-    "webpack-dev-server": "^2.5.1"
+    "webpack-dev-server": "^2.9.4",
+    "webpack-merge": "^4.1.1"
   },
   "optionalDependencies": {
     "phantomjs-prebuilt": "^2.1.0"
   },
   "dependencies": {
+    "@rails/webpacker": "^3.0.2",
     "babel-polyfill": "^6.26.0",
     "brace": "^0.10.0",
     "c3": "^0.4.11",
-    "datatables.net": "~1.10.12",
-    "datatables.net-bs": "~1.10.12",
+    "coffeescript": "^2.0.2",
+    "datatables.net": "1.10.15",
+    "datatables.net-bs": "1.10.15",
```

Comment:
were you able to use datatables with yarn? I got https://github.com/DataTables/Dist-DataTables-Bootstrap/pull/2

Reply:
I had some problems and couldn't really understand the reason for that.
At the end, I dived into the code and found some solution/hack but I will give it a second look while considering your issue.

Reply:
I am not sure what causing it, it seems the versions are fine at the `yarn.lock`.
If upgrade to 1.10.16 i am getting errors.

Reply:
i think this is exactly https://github.com/DataTables/Dist-DataTables-Bootstrap/pull/2

Reply:
OK, i checked it again and npm resloving `1.10.15` while yarn resolving `1.10.16`.
When  using`npm` i don't need to change the `package.json` and the `bundle_datatables.js`.

Do you have any suggestions?

Reply:
read the PR with reference to yarn issue..... tl/dr - merge the pr :)

Reply:
you mean wait/put pressure on `datatables.net-bs` to merge you pr?
or switch back to npm? I didn't realized foreman used to work with `yarn` before.

Reply:
@ohadlevy - updating jquery to v3 solving the issue!
I am working on a separate pr to update it.

Anyway, we will be able to replace those data tables soon after https://github.com/theforeman/foreman/pull/5024 will merge.